src/components/Scene.js

```diff
@@ -61,10 +61,10 @@ class Scene extends Component {
   renderPods() {
     const {queries, removePod} = this.props;
     return queries.all
-      .map((query, i) => (
```

Comment:
It causes a bug because the index of the pods changes as they are deleted/inserted.